packages/desktop-client/e2e/transactions.test.js

```diff
@@ -63,7 +63,7 @@ test.describe('Transactions', () => {
       await expect(autocomplete).toMatchThemeScreenshots();
 
       // Select the active item
-      await page.getByRole('button', { name: 'Clothing' }).click();
+      await page.getByTestId('Clothing-category-item').click();
```

Comment:
Personally, I prefer `getByRole` with the user-visible text over grabbing elements by something the user can't see, in order to keep our tests behaving similarly to a user. Did the existing selector fail after the refactor?

As I say, it's a preference, so ship it as you see fit.

Reply:
For some reason, the test is hanging on this `getByRole` call so I had to change it. I'm happy to revert it back to use `getByRole` if anybody can get it to work again.

Reply:
Not worth holding it up for (although we're in code freeze at the moment).